article.css

```
@@ -0,0 +1,143 @@
+#section1
+{
+    display: flex;
+    justify-content: space-between;
+}
+
+#article1
+{
+    border: 1px solid #F2F2F2;
+    margin-left: 15px;
+    width: 1045px;
+    background-color: #F2F2F2;
+    border-radius: 10px;
+    height: 645px;
+    margin-top: 26px;
+}
+
+#hosting
+{
+    font-size: 18px;
+    margin-left: 40px;
+    margin-top: 50px;
+    margin-bottom: 50px;
+}
+
+.hostingS
+{
+    display: flex;
+    margin-left: -55px;
+    margin-top: -95px;
+}
+
+.colomN
+{
+    display: block;
+}
+
+.squares
+{
+    border: 1px solid white;
+    width: 225px;
+    background-color: white;
+    padding: 2px;
+    border-radius: 10px;
+    box-shadow: 2px 2px 5px grey;
+    height: 155px;
+    margin-left: 115px;
+    margin-top: 95px;
+}
+
+ul
+{
+    list-style-type: none;
+    justify-content: space-around;
+}
+
+.images
+{
+    width: 225px;
+    height: 105px;
+    border-radius: 10px 10px 0px 0px;
+    margin-top: -16px;
+    margin-left: -40px;
+}
+
+.names
+{
+    margin-left: -35px;
+    font-size: 14px;
+    font-weight: bold; 
+}
+
+.nights
+{
+    font-size: 12px;
+    margin-left: -35px;
+}
+
+.rating
+{
+    color: #F2F2F2;
+    font-size: 11px;
+    margin-left: 4px;
+    margin-top: -15px;
+}
+
+.stars
+{
+    float: left;
+}
+
+.stars2
+{
+    margin-right: -410px;
+}
+
+.stars3
+{
+    margin-right: -125px;
+}
+
+.stars i,
```

this can be summurised like that

```suggestion
[class^="stars"] i
```
which normally should select every elements with a class name starting by stars. https://www.geeksforgeeks.org/wildcard-selectors-and-in-css-for-classes/

---

I didn't try this method I'll try it now, should be easier

---

let me know when you are done.

---

Done! It's working pretty well !